.github/workflows/ci.yml

```diff
@@ -165,6 +163,8 @@ jobs:
     steps:
       - uses: 'actions/checkout@v4'
         name: 'Checkout'
+        with:
+          ref: ${{ inputs.refName != '' && inputs.refName || github.ref_name }}
```

Comment:
We will be better off falling back to `''` so the checkout action keeps doing its internal magic, which is currently stable.

Reply:
There are more similar places in this workflow that this comment covers, but I'll avoid duplicating them yet.

Reply:
Good point, it was failing on prs anyway with ref_name. Updated with [5b7700](https://github.com/woocommerce/woocommerce/pull/56366/commits/25b770027042c30e36af1f2cf57a0b315073ec8f)